sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOverrides.scala

```diff
@@ -2858,6 +2858,22 @@ object GpuOverrides extends Logging {
           GpuArrayRemove(lhs, rhs)
       }
     ),
+    expr[MapFromArrays](
+      "Creates a new map from two arrays",
+      ExprChecks.binaryProject(
+        TypeSig.MAP.nested(TypeSig.commonCudfTypes),
+        TypeSig.MAP.nested(TypeSig.commonCudfTypes),
```

Comment:
nit: Because this like is for what Spark supports I think the type needs to be
```
TypeSig.MAP.nested(TypeSig.all)
```

Because that way it matches what we say are the input types below.